components/formats-gpl/src/loci/formats/in/SDTReader.java

```diff
@@ -98,14 +116,7 @@ public void setIntensity(boolean intensity) {
     this.intensity = intensity;
   }
 
-  /**
-   * Toggles whether the reader should pre-load
-   * data for increased performance.
-   */
-  public void setPreLoad(boolean preLoad) {
-    this.preLoad = preLoad;
-  }
-
```

Comment:
As it is public API, could this please be deprecated instead of being removed outright?